Anlab.Core/Domain/MailMessage.cs

```diff
@@ -28,5 +28,9 @@ public MailMessage()
         public DateTime? SentAt { get; set; }
 
         public DateTime CreatedAt { get; set; }
+
+        public Order Order { get; set; }
```

Comment:
@jSylvestre I didn't do this on purpose, I'm not sure we'll want to have every message associated with an order.  What if we want to send someone a message about their clientID or payment information?